fe/.eslintrc.js

```diff
@@ -26,10 +32,37 @@ module.exports = {
     // Ignore `import` statements with Webpack `@` alias in path
     'import/no-unresolved': ['error', { ignore: ['^@'] }],
 
+    // Set maximum line length to 120 columns if line doesn't contain is not url, string, template literal or regex
```

Comment:
```suggestion
    // Set maximum line length to 120 characters. Lines containing URLs, strings, template literals or regular
    // expressions will be ignored.
```

Reply:
> ... if line doesn't contain is not url, ...

I was unable to translate this part. Notify me if my version of the comment is wrong.